src/proposals/20251002/SparkEthereum_20251002.sol

```diff
@@ -0,0 +1,292 @@
+                (assets)
+            )
+        );
+
+        MainnetController(Ethereum.ALM_PROXY).revokeRole(
+            IALMProxy(Ethereum.ALM_PROXY).CONTROLLER(),
+            Ethereum.SPARK_PROXY
+        );
+
+        // Add transferAsset Rate Limit for SYRUP
+        SLLHelpers.setRateLimitData(
+            RateLimitHelpers.makeAssetDestinationKey(
+                MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_ASSET_TRANSFER(),
+                SYRUP,
+                Ethereum.ALM_OPS_MULTISIG
+            ),
+            Ethereum.ALM_RATE_LIMITS,
+            200_000e18,
+            200_000e18 / uint256(1 days),
+            18
+        );
+
+        // Transfer Share of Ethena Direct Allocation Net Profit to Grove
+        IERC20(Ethereum.USDS).transfer(GROVE_SUBDAO_PROXY, AMOUNT_TO_GROVE);
+
+        // Spark Foundation
+        IERC20(Ethereum.USDS).transfer(Ethereum.SPARK_FOUNDATION, AMOUNT_TO_SPARK_FOUNDATION);
+
+        // Configure Symbiotic Instance
+        _configureSymbiotic();
+
+        // Withdraw USDS and DAI Reserves from SparkLend
+        address[] memory aTokens = new address[](2);
+        aTokens[0] = Ethereum.DAI_SPTOKEN;
+        aTokens[1] = Ethereum.USDS_SPTOKEN;
+
+        _transferFromSparkLendTreasury(aTokens);
+    }
+
+    function _configureSymbiotic() internal {
+        address NETWORK   = Ethereum.SPARK_PROXY;
+        address OWNER     = Ethereum.SPARK_PROXY;
+        address OPERATOR  = Ethereum.SPARK_PROXY;
+
+        bytes32 subnetwork = bytes32(uint256(uint160(NETWORK)) << 96 | 0);  // Subnetwork.subnetwork(network, 0)
+
+        INetworkRestakeDelegator delegator = INetworkRestakeDelegator(NETWORK_DELEGATOR);
+        INetworkRegistry networkRegistry   = INetworkRegistry(NETWORK_REGISTRY);
+        IOperatorRegistry operatorRegistry = IOperatorRegistry(OPERATOR_REGISTRY);
+        IVetoSlasher slasher               = IVetoSlasher(VETO_SLASHER);
+
+        // --- Step 1: Do configurations as network, DO NOT SET middleware, max network limit, and resolver
+        networkRegistry.registerNetwork();
+        delegator.setMaxNetworkLimit(0, type(uint256).max);
+        slasher.setResolver(0, OWNER, "");
+
+        // --- Step 2: Configure the network and operator to take control of stake as the vault owner
+        delegator.setNetworkLimit(subnetwork, type(uint256).max);
+        delegator.setOperatorNetworkShares(
+            subnetwork,
+            OPERATOR,
+            1e18  // 100% shares
+        );
+        delegator.setHook(RESET_HOOK);
+        delegator.grantRole(delegator.OPERATOR_NETWORK_SHARES_SET_ROLE(), RESET_HOOK);
+
+        // --- Step 3: Opt in to the vault as the operator
+        operatorRegistry.registerOperator();
+        IOptInService(delegator.OPERATOR_NETWORK_OPT_IN_SERVICE()).optIn(NETWORK);
+        IOptInService(delegator.OPERATOR_VAULT_OPT_IN_SERVICE()).optIn(STAKED_SPK_VAULT);
+    }
+
+    function _configureVaultsV2(
```

Comment:
I also agree seeding the vault (presumably at the end of this function, but feel free to a better location) is a good idea